aten/src/ATen/cuda/CUDASparse.h

```diff
@@ -34,8 +34,7 @@
 
 // BSR triangular solve functions were added in hipSPARSE 1.11.2 (ROCm 4.5.0)
 #if defined(CUDART_VERSION) ||                            \
-    (defined(USE_ROCM) && (hipsparseVersionMajor >= 1) && \
-     (hipsparseVersionMinor >= 11) && (hipsparseVersionPatch >= 2))
+      (defined(USE_ROCM) && ROCM_VERSION >= 40500 )
```

Comment:
Is it possible to install ROCm 4.5 with an older hipSPARSE version?

Reply:
hipSparse is part of ROCm and we dont support mix and match packages between different ROCm releases